tests/integration_tests/config/generate_default_metadata_to_json_file.yml

```diff
@@ -0,0 +1,9 @@
+CTA_SIMPIPE:
+  APPLICATION: simtools-generate-default-metadata
+  TEST_NAME: test-json
+  CONFIGURATION:
+    OUTPUT_FILE: test.json
+    OUTPUT_PATH: simtools-tests
+  INTEGRATION_TESTS:
+    - OUTPUT_FILE: test.json
+    - FILE_TYPE: json
```

Comment:
I tested this implementation by running the application in the container with 
```
simtools-generate-default-metadata --output_file tst.json --output_path .
```
but I get an error that 
```
generate_default_metadata: error: the following arguments are required: --schema
```

I remember that with the config files we can not assure that all necessary arguments were passed, but I suggest adding the `--schema` argument into the config file to really test the application.

Reply:
We are absolutely sure that all parameters are passed with the `--config` parameters. Do you have an example where this is not the case?

Are you sure you did also a `pip install -e` after checking the branch out? It works for me and the integration tests on github are also working.